ledger/src/tests.rs

```diff
@@ -2569,8 +2581,21 @@ fn test_no_rewards_after_limit_height() {
         ledger.prepare_advance_to_next_beacon_block(&private_key, vec![], vec![valid_solution], vec![], rng).unwrap();
     ledger.advance_to_next_block(&next_block_with_solution).unwrap();
 
-    // Check that there are no rewards in the block.
-    assert!(next_block.ratifications().is_empty());
+    // Check that the block and puzzle rewards are 0.
+    assert!(!next_block.ratifications().is_empty());
+    let ratifications: Vec<_> = next_block.ratifications().iter().collect();
+    match ratifications[0] {
+        Ratify::BlockReward(block_reward) => {
+            assert_eq!(*block_reward, 0);
+        }
+        _ => panic!("Expected a block reward ratification"),
+    }
+    match ratifications[1] {
+        Ratify::PuzzleReward(puzzle_reward) => {
```

Comment:
I assume the overhead of keeping block rewards of 0 is minimal compared to not containing them at all?

Reply:
yea exactly, our `post-ratify` logic currently expects a `Ratify::BlockReward` object (even if it is 0) to update the committee mapping.